plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -595,9 +595,18 @@ def _is_retryable(error):
 
 _SANITIZE_LABEL_PATTERN = re.compile(r"[^a-z0-9_-]")
 
+_VALIDATE_LABEL_LENGTH_LIMIT = 63
+
 
 def _sanitize_label(value: str) -> str:
     """Return a legal value for a BigQuery label."""
     value = value.strip().lower()
     value = _SANITIZE_LABEL_PATTERN.sub("_", value)
-    return value
+    value_length = len(value)
+    if value_length > _VALIDATE_LABEL_LENGTH_LIMIT:
+        error_msg = (
+            f"Current label length {value_length} is greater than length limit: {_VALIDATE_LABEL_LENGTH_LIMIT} | Current sanitized label: {value}"
```

Comment:
Our style formatter (flake8) says this line is too long, which is why one of the CI tests is failing:
```
plugins/bigquery/dbt/adapters/bigquery/connections.py:608:100: E501 line too long (146 > 99 characters)
```
It's also a bit long on the CLI. How about making this two lines?
```suggestion
            f"Job label length {value_length} is greater than length limit: " \
            f"{_VALIDATE_LABEL_LENGTH_LIMIT}\n" \
            f"Current sanitized label: {value}"
```